client/src/components/EmojiPicker.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
```

Comment:
There's one general complicated part in EmojiPicker which is the usage of emojilist.js vs react-emoji. They contain different sets of emojis and as such it looks like you need to figure out at runtime which ones are supported by react-emoji. To make it easier, let's combine the two in a way that doesn't require runtime checks. See details below.